src/clj_icat_direct/queries.clj

```diff
@@ -629,6 +637,10 @@
      [(analyze "objs")]
      [(mk-temp-table "file_avus" (mk-obj-avus "SELECT data_id FROM objs"))]
      [(analyze "file_avus")]
+     [(mk-temp-table "colls" (mk-colls-in-coll parent-path))]
+     [(analyze "colls")]
+     [(mk-temp-table "coll_avus" (mk-obj-avus "SELECT coll_id FROM colls"))]
+     [(analyze "coll_avus")]
```

Comment:
So what I was initially thinking here, and maybe you tried this or maybe it merits trying, is actually having `file_avus` and `coll_avus` be one table that's just all the relevant AVUs for this listing. Then it's only one temporary table and one analyze that needs to happen. I'm also unsure that the `colls` temporary table is likely to help much here -- doing `mk-obj-avus` with a `SELECT coll_id FROM r_coll_main WHERE parent_coll_name = "$1"` might be more efficient. Certainly doing an analyze on the `colls` temporary table isn't helping anything, because `coll_avus` will need to sequential scan it anyway.

Reply:
Though it does occur to me that if the `colls` temporary table is making things more efficient, we could probably also use it in the `mk-folders-in-folder` query instead of `r_coll_main` directly as well. Not much sense pulling those out twice.

And I guess we can add the `c.coll_type != 'linkPoint'` whether we do it inline to `mk-obj-avus` or in a temporary table, too.

Reply:
It appears that removing the extra temporary table also removed the additional extra overhead for non-problem accounts:

| Path | Entity Type | User | Duration | Number of Entries |
| ---- | ----------- | ---- | -------- | ----------------- |
|  /iplant/home  |  :any  |  sarahr  |  3737  |  161 |
|  /iplant/home  |  :any  |  cosimichele  |  4659  |  166 |
|  /iplant/home  |  :any  |  nirav  |  5552  |  206 |
|  /iplant/home  |  :folder  |  sarahr  |  4030  |  161 |
|  /iplant/home  |  :folder  |  cosimichele  |  4958  |  166 |
|  /iplant/home  |  :folder  |  nirav  |  5897  |  206 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :any  |  sarahr  |  47  |  469 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :any  |  cosimichele  |  47  |  469 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :any  |  nirav  |  52  |  469 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :folder  |  sarahr  |  32  |  465 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :folder  |  cosimichele  |  35  |  465 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :folder  |  nirav  |  40  |  465 |

Reply:
And here are the numbers after also removing it from `mk-paged-folders-in-folder`

Reply:
It appears to be running slightly slower with the combined AVUs table, but the difference is largely negligible:

| Path | Entity Type | User | Duration | Number of Entries |
| ---- | ----------- | ---- | -------- | ----------------- |
|  /iplant/home  |  :any  |  sarahr  |  3842  |  161 |
|  /iplant/home  |  :any  |  cosimichele  |  4766  |  166 |
|  /iplant/home  |  :any  |  nirav  |  5670  |  206 |
|  /iplant/home  |  :folder  |  sarahr  |  3722  |  161 |
|  /iplant/home  |  :folder  |  cosimichele  |  4660  |  166 |
|  /iplant/home  |  :folder  |  nirav  |  5557  |  206 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :any  |  sarahr  |  47  |  469 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :any  |  cosimichele  |  45  |  469 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :any  |  nirav  |  50  |  469 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :folder  |  sarahr  |  28  |  465 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :folder  |  cosimichele  |  31  |  465 |
|  /iplant/home/shared/iplantcollaborative/example_data  |  :folder  |  nirav  |  34  |  465 |